projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -744,22 +743,33 @@ public static function get_red_bubble_alerts() {
 	 * @return array
 	 */
 	public static function check_for_broken_modules() {
-		$broken_modules              = array(
+		$is_user_connected = ( new Connection_Manager() )->is_user_connected() || ( new Connection_Manager() )->has_connected_owner();
+		$is_site_connected = ( new Connection_Manager() )->is_connected();
```

Comment:
could be cleaner if we define Connection_Manager as variable first:
`$manager = new Connection_Manager();`

Just a minor comment

Reply:
Nice improvement, I'll update that real quick